pkg/clustermesh/clustermesh.go

```diff
@@ -73,6 +76,43 @@ type Configuration struct {
 	NodesSharedKeyDeleteDelay *time.Duration
 }
 
+func SetClusterConfig(clusterName string, config *cmtypes.CiliumClusterConfig, backend kvstore.BackendOperations) error {
+	key := path.Join(kvstore.ClusterConfigPrefix, clusterName)
+
+	val, err := json.Marshal(config)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	_, err = kvstore.Client().UpdateIfDifferent(ctx, key, val, false)
```

Comment:
Shouldn't we update with a "lease"? This way the configuration would be removed automatically once the lease expired.

Reply:
Oops, thank you. I fixed.